float64 az_speed
float64 el_speed
float64 dist
bool limit